lib/_debugger.js

```diff
@@ -995,14 +960,16 @@ Interface.prototype.debugEval = function(code, context, filename, callback) {
     }
 
     // Request object by handles (and it's sub-properties)
-    client.mirrorObject(res, 3, function(err, mirror) {
+    client.mirrorObject(res, 3, function(error, mirror) {
+      if (error) {
+        // ignore
+      }
```

Comment:
Better tweak the `no-unused-vars` option for args instead of doing it like this.